minecraft/data.json

```diff
@@ -47,6 +47,22 @@
       "type": "host"
     }
   },
+  {
+    "name": "minecraft-neoforge",
+    "variables": {
+      "memory": "2048",
+      "ip": "0.0.0.0",
+      "port": "25565",
+      "eula": true,
+      "version": "1.20.1",
```

Comment:
Might be better to just use 1.20.4 or something. Neo even suggests Forge for 1.20.1 instead. And, the actual changes are in the .2 onward.